RELEASING.md

```diff
@@ -289,6 +289,11 @@ Perform the following steps on a local branch off of the latest master.
    git commit -m "Updates for release vX.Y.Z"
    ```
 
+1. Push your branch and open a pull request. Get it reviewed and wait for it to pass CI.
```

Comment:
I think strictly speaking we'll want to split this step.

It should be:

1. Submit PR, get it approved
2. Build the release
3. Publish the release
4. Merge the PR

Reply:
That's more or less what it says, just with a bit more detail. I moved the section on merging to the end tho.